saleor/graphql/account/mutations/account/account_request_deletion.py

```diff
@@ -46,7 +46,11 @@ class Meta:
             WebhookEventInfo(
                 type=WebhookEventAsyncType.NOTIFY_USER,
                 description="A notification for account delete request.",
-            )
+            ),
+            WebhookEventInfo(
+                type=WebhookEventAsyncType.ACCOUNT_DELETE_REQUESTED,
+                description="An account delete requested.",
```

Comment:
```suggestion
                description="An account delete was requested.",
```

Reply:
or
`description="A request to delete the account has been made."`